dj_beneficiary/models.py

```diff
@@ -103,20 +100,13 @@ class FacilityType(models.Model):
     def __str__(self):
         return self.name.title()
 
-class Facility(models.Model):
+class BaseFacilityAbstract(models.Model):
```

Comment:
`AbstractFacilityModel`

Reply:
This won't be necessary since the Facility model won't be abstract.